flytekit/extras/tasks/shell.py

```diff
@@ -98,6 +99,30 @@ def interpolate(
 T = typing.TypeVar("T")
 
 
+def _run_script(script) -> typing.Tuple[int, str, str]:
+    """
+    Run script as a subprocess and return the returncode, stdout, and stderr.
+
+    While executing the su process, stdout of the subprocess will be printed
+    to the current process stdout so that the subprocess execution will not appear unresponsive
+
+    :param script: script to be executed
+    :type script: str
+    :return: tuple containing the process returncode, stdout, and stderr
+    :rtype: typing.Tuple[int, str, str]
+    """
+    process = subprocess.Popen(script, stdout=subprocess.PIPE, stderr=subprocess.PIPE, bufsize=0, shell=True, text=True)
+
+    # print stdout so that long-running subprocess will not appear unresponsive
```

Comment:
qq: Currently, the pod's log didn't show any stdout and stderr?

Reply:
Currently it does. However this PR capture the stdout thus need to emit it manually.